CMR/python/cmr/auth/token.py

```diff
@@ -1,98 +1,38 @@
-"""
-NASA EO-Metadata-Tools Python interface for the Common Metadata Repository (CMR)
-
-    https://github.com/nasa/Common-Metadata-Repository/
+# NASA EO-Metadata-Tools Python interface for the Common Metadata Repository (CMR)
+#
+#     https://github.com/nasa/Common-Metadata-Repository/
```

Comment:
This isn't something a client would glean usage information from right? I would argue the link here should be: https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html